docs/3-déploiement.md

```diff
@@ -4,36 +4,33 @@ RDV-Solidarités est hébergé chez [Scalingo](https://scalingo.com/fr/datacente
 
 | Instance | domaine | branche | notes |
 | -------- | ------- | ------- | ----- |
-| production-rdv-solidarites | https://www.rdv-solidarites.fr | production | - |
+| production-rdv-solidarites | https://www.rdv-solidarites.fr | production | review apps activées |
 | demo-rdv-solidarites | https://demo.rdv-solidarites.fr | production | - |
-| recette-rdv-solidarites | https://recette.rdv-solidarites.fr | recette | review apps activées |
 
+Les Pull Requests sont mises en production automatiquement dès qu’elles sont mergées sur la branche `production`.
 
-Pour permettre aux personnes référentes 
-
-* d'être informer avant les mises en production ;
-* de pouvoir tester les changements apportés ;
-
-nous utilisons un mode de livraison hebdomadaire. Nous avons aussi une procédure d'urgence pour pouvoir apporter des modifications très rapidement en production. C'est une procédure exceptionnelle.
 
 ### Cas nominal
 
-* L'équipe réalise une modification du service sur une **branche spécifique**. Cette modification est déployée sur un environnement de test très simple \(_review app_\) pour permettre à l'équipe de vérifier le bon fonctionnement des changements.
-* Lorsque l'équipe pense que la modification du service correspond à ce qui a été défini, elle _merge_ les modifications dans une branche « recette ». Cette modification déployée sur l'environnement « recette » à l'adresse [https://recette.rdv-solidarites.fr](https://recette.rdv-solidarites.fr).
-* Le mardi présentation des nouveautés, sur l'environnement de recette. Cet environnement est accessible aux référentes. Avec des données fictives. Les configuration et cas de test pourront être préservé d'une livraison à l'autre, un peu comme pour l'environnement de démo.
-* Le jeudi, sauf demande particulière, nous faisons le « merge » la banche de recette sur la branche de production. Seul les éléments démontré le mardi seront mis en production. Nous allons utiliser un tag, le lundi soir ou mardi matin, avec la date pour signaler un point de livraison. La branche de production est automatiquement déployée sur l'environnement de production.
+Les changements qui ne demandent pas de démo ou de communication avec l’équipe des référentes sont mergées et mises en production directement. 
```

Comment:
J’attends vos commentaires sur la forme et le fond de ces paragraphes :)

Reply:
Je définirais peut-être que ce sont « les changements impactants (ou modifiant ?) l'usage du service doivent être montré en démo aux référentes pour les tenir informés et vérifier qu'il n'y a pas de véto avant d'être mis en production. Les autres changements peuvent partir en production directement ». Mais c'est plus long ... :thinking: